src/main/resources/MessageBundle.properties

```diff
@@ -200,7 +200,7 @@ writing_line_to_file=Writing line [{}] to [{}]
 
 #for BagWriter.java
 writing_payload_files=Writing payload files.
-upsert_payload_oxum=Upserting payload-oxum.
+upsert_payload_oxum=Inserting payload-oxum.
```

Comment:
No, upserting is correct. It stands for inserting or updating.

Reply:
Sorry and thanks for extending my English vocabulary. 👍

Reply:
👍

Reply:
Who are the intended audience for this message? If not, I'd be inclined to use something like “updating” since “upsert” is generally only familiar to people who work with databases a lot.

Reply:
It is a DEBUG message, which I generally wrote for other programmers.